packages/googlechrome.vm/googlechrome.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>googlechrome.vm</id>
-    <version>0.0.0.20241106</version>
+    <version>0.0.0.20241113</version>
     <authors>Google LLC.</authors>
     <description>Chrome is a popular web browser.</description>
     <dependencies>
-      <dependency id="common.vm" version="0.0.0.20241106" />
-      <dependency id="vcbuildtools.vm" />
```

Comment:
Ah, I didn't realize that was just added in your recent PR for the signature checking:
https://github.com/mandiant/VM-Packages/issues/1144#issuecomment-2459017651

I'll add that back in.

Reply:
Ah true, this is now needed because of the use of `signtool.exe`. I'll add a comment as a separate PR because if I didn't understand it a week after I implemented it, likely it is going to be removed often...